backend/routes/postReaction.py

```
@@ -15,6 +15,7 @@
 from zoneinfo import ZoneInfo
 from crud.notification import create_notification
 from schemas.notification import NotificationCreate
+from services.reaction import *
```

Ruff (0.8.2)

`from services.reaction import *` used; unable to detect undefined names

(F403)